python_listener/client.py

```diff
@@ -38,27 +53,51 @@
                 "instructions"
             ]:
                 if instruction["programIdIndex"] == audius_program_index:
-                    hex_data = binascii.hexlify(
-                        bytearray(list(base58.b58decode(instruction["data"])))
-                    )
+                    signed_msg = base58.b58decode(instruction['data'])[65:].decode()
+                    print(signed_msg)
+
+    # Check tx for tracklistencount program, log statement if found
+    transaction2_slot = transaction2["result"][0]["slot"]
+    print(f'CREATE_AND_VERIFY: {transaction2_slot} > {slot_from}')
+    # Clarify why this check was > and not >=, is this on purpose?
+    # if transaction2_slot >= slot_from:
+    if transaction2_slot >= slot_from:
```

Comment:
For some reason, making this >= was required in order to process any operation?

Reply:
@ysavchenko - is there some other processing required here?

Reply:
On the original code it was the way to ensure the same transaction does not get processed twice. `slot` is a block number, so when getting the last transaction for the address we check if it was not already processed (by saving `slot_from`, which is the block for transaction processed earlier)

In your code you have two transactions, but save their block to the same variable and you get a conflict. What you should do is have different `slot_from` variables, one for the first transaction and one for the second.

Reply:
wow...this makes so much sense, of course it is getting overridden!